### PR TITLE
Small XP gain changes

### PR DESCRIPTION
**Pull request by Nagath**
coded by theanax

Added an additional point to gain on kill and on kill with taking damage that should effectively up the HP gain rate a bit.

Changed the newbie bonus XP to scale with whatever the angel setting is rather then stopping at 25.

### CODE REVIEW

File: kod/object/active/holder/nomoveon/battler/player.kod [hunk 1]
```diff
@@ -7663,7 +7663,7 @@ messages:
          }
 
          % Give newbies a little bonus.
-         if piBase_Max_health < 26 and random(20,bound(piBase_Max_health,20,25)) = 20
+         if piBase_Max_health = PKILL_ENABLE_HP
```
Comment (skittles1) [inline kod/object/active/holder/nomoveon/battler/player.kod]: This only gives the bonus if the player's max heal is equal to PKILL_ENABLE_HP. We have this on 103:

     if piBase_Max_health < Send(Send(SYS,@GetSettings),@GetPKillEnableHP)

So you would want to use:

    if piBase_Max_health < PKILL_ENABLE_HP

**Comment by Nagath** [comment]
I adjustet it the same way. Its proven it works well like this on 103 ^^

**Comment by Koveras** [comment]
+1, this would especially help in group scenarios since like this the bonus for getting hit is "only" 50%, rather than 100%.

**Comment by davidklaw** [comment]
Seems harmless, and in some small way offers new players a slightly more encouraging onboarding experience.

File: kod/object/active/holder/nomoveon/battler/player.kod [hunk 2]
```diff
@@ -7663,7 +7663,7 @@ messages:
          }
 
          % Give newbies a little bonus.
-         if piBase_Max_health < 26 and random(20,bound(piBase_Max_health,20,25)) = 20
+         if piBase_Max_health = < Send(Send(SYS,@GetSettings),@GetPKillEnableHP)
```
Comment (davidklaw) [inline kod/object/active/holder/nomoveon/battler/player.kod:7666]: @Meridian59 We missed this `= <` and introduced a compilation error. I'm putting together a new PR.

**Comment by akirmse** [comment]
Whoops, I should have seen this.  The pre-submit action will find such
things in the future.  I wasn't able to find a way to run actions against
PRs that predate them.

**Pull request merged**